src/encoded/static/components/browse/columnExtensionMap.js

```diff
@@ -55,7 +55,7 @@ export const columnExtensionMap = _.extend({}, basicColumnExtensionMap, {
     'lab.display_title' : {
         'title' : "Lab",
         'widthMap' : { 'lg' : 200, 'md' : 180, 'sm' : 160 },
-        'render' : labDisplayTitleRenderFxn
+        'render' : LabColumn
```

Comment:
I'm not sure this would work ^ (pls double-check), I think may need to do ~:

```javascript
'render' : function(result, props){
    const { lab, submitted_by : { display_title : submitterTitle } = {} } = result;
    if (lab) {
        return <LabColumn {...{ lab, submitterTitle } />;
    }
    return null;
}
```

Reply:
@alexkb0009  Uğur updated the render function.